releasenotes/notes/deprecate-dd-tracer-configs-b05936f5ab56d128.yaml

```diff
@@ -0,0 +1,20 @@
+---
+features:
+  - |
+    This change introduces new config keys for enabling and configuring 
+    partial span flushing. 
+
+    This change does not introduce any additional functionality. It only 
+    deprecates DD_TRACER... config keys and replaces it an equivalent 
+    DD_TRACE... configuration while maintaining backwards compatibility.
+upgrade:
+  - |
+    Replace DD_TRACER_PARTIAL_FLUSH_ENABLED with DD_TRACE_PARTIAL_FLUSH_ENABLED
+    Replace DD_TRACER_PARTIAL_FLUSH_MIN_SPANS with DD_TRACE_PARTIAL_FLUSH_MIN_SPANS
+deprecations:
+  - |
+    Adds deprecatation warning for the DD_TRACER_PARTIAL_FLUSH_ENABLED and 
+    DD_TRACER_PARTIAL_FLUSH_MIN_SPANS configurations. 
+
+    With the release of version 1.0.0 these configuration will no longer be 
```

Comment:
Can just merge this sentence with the one above and simplify to just "These options will be removed in version 1.0"